proto/buf.yaml

```diff
@@ -7,6 +7,16 @@ breaking:
 lint:
   use:
     - DEFAULT
+    - COMMENTS
+    - FILE_LOWER_SNAKE_CASE
+  except:
+    - UNARY_RPC
+    - COMMENT_FIELD
+    - SERVICE_SUFFIX
+    - PACKAGE_VERSION_SUFFIX
+    - RPC_REQUEST_STANDARD_NAME
```

Comment:
Do these lints need to be overridden across the board, or only for legacy packages?
In the `ignore_only` object below, there is a pattern on how to only waive lints for the beta packages which did not follow best practices.

Reply:
See https://github.com/cometbft/cometbft/pull/1703/commits/fe990e73983223f3c78db79a10666ba916668724

We only need to exclude `COMMENT_FIELD` **across the board**.